cmd/run.go

```diff
@@ -22,16 +22,25 @@ var runCmd = &cobra.Command{
 			return err
 		}
 
+		force_jitsi, err := cmd.Flags().GetBool("jitsi")
+		if err != nil {
+			return err
+		}
+
 		manual := false
 		if url == "" {
 			manual = true
 		}
 
-		// TODO: When additional video conference platforms are added, make this
-		// url evaluation more sophisticated.
+		var scrape_func = scrape.GetParticipantsZoom
```

Comment:
I recommend declaring a Scraper type in the scrape package.

Something like:
```
type Scraper func(url string, refreshInterval int, data *tui.Data, pw *playwright.Playwright) error
```

Then you can use that in `cmd` like so:
```
var scraper Scraper
```
-- instead of having it default to zoom.

Reply:
I also recommend avoiding snake_case in Golang. I do that all the time when I'm pivoting from Python, heh. You can probably just call the variable `scraper`.

Reply:
See, I am mainly doing rust and python atm, so that's where the preference to snake_case comes from :P